akka-cluster-sharding-typed/src/test/scala/docs/akka/cluster/sharding/typed/AccountExampleWithCommandHandlersInState.scala

```diff
@@ -131,6 +131,7 @@ object AccountExampleWithCommandHandlersInState {
         throw new IllegalStateException(s"unexpected event [$event] in state [ClosedAccount]")
     }
 
+    // used for sharding, typically in `init`-- for an example, see AccountExampleSpec.scala line 47 in this folder
```

Comment:
Might be good to clarify what TypeKey is for. Since most people will be looking at this example from the docs, perhaps it's not very helpful to refer to `AccountExampleSpec.scala` though. Perhaps something like:

```suggestion
    // when used with sharding, this TypeKey can be used in `sharding.init` and `sharding.entityRefFor`:
```

Reply:
good for me as well, updated it accordingly